src/ParametricOptInterface.jl

```diff
@@ -7,8 +7,10 @@ const MOI = MathOptInterface
 
 const PARAMETER_INDEX_THRESHOLD = 1_000_000_000_000_000_000
 const SUPPORTED_SETS =
-    (MOI.LessThan{Float64}, MOI.EqualTo{Float64}, MOI.GreaterThan{Float64})
-const SUPPORTED_VECTOR_SETS = (MOI.Nonnegatives, MOI.SecondOrderCone)
+    (MOI.LessThan{Float64}, MOI.EqualTo{Float64}, MOI.GreaterThan{Float64},
+    MOI.Integer, MOI.ZeroOne)
```

Comment:
I think that it is not necessary to add `Integer` and `ZeroOne` because these guys are `VariableIndex-in-Set` and we already prevent users from adding a parameter in set.

Reply:
Also I am working on trying to eliminate the need for `SUPPORTED_SETS`.

Reply:
Just test `ZeroOne`, then?

Reply:
Only real variables are allowed in those cases, so we don't need to make tests.